src/Uno.Templates/content/unoapp/MyExtensionsApp.1/Properties/PublishProfiles/win-x86.pubxml

```diff
@@ -12,7 +12,8 @@ https://go.microsoft.com/fwlink/?LinkID=208121.
 		<PublishSingleFile>False</PublishSingleFile>
 		<PublishReadyToRun Condition="'$(Configuration)' == 'Debug'">False</PublishReadyToRun>
 		<PublishReadyToRun Condition="'$(Configuration)' != 'Debug'">True</PublishReadyToRun>
-		<!-- Note: Trimming disabled by default as there may still be an issues with PublishTrimmed support: https://github.com/microsoft/CsWinRT/issues/373 -->
+		<PublishTrimmed Condition="'$(Configuration)' == 'Debug'">False</PublishTrimmed>
+		<PublishTrimmed Condition="'$(Configuration)' != 'Debug'">True</PublishTrimmed>
```

Comment:
Why are we changing this as part of this PR?

Reply:
It was just merge conflict as the trimming PR was merged first, now rebased 👍